README.md

```diff
@@ -694,6 +694,37 @@ OPENAI_TOKEN=XXX go test -v -tags=integration ./api_integration_test.go
 
 If the `OPENAI_TOKEN` environment variable is not available, integration tests will be skipped.
 
+## Frequently Asked Questions
+
+### Why don't we get the same answer when specifying a temperature field of 0 and asking the same question?
+
+Even when specifying a temperature field of 0, it doesn't guarantee that you'll always get the same response. Several factors come into play.
+
+1. Go OpenAI Behavior: When you specify a temperature field of 0 in Go OpenAI, the omitempty tag causes that field to be removed from the request. Consequently, the OpenAI API applies the default value of 1.
+2. Token Count for Input/Output: If there's a large number of tokens in the input and output, setting the temperature to 0 can still result in non-deterministic behavior. In particular, when using around 32k tokens, the likelihood of non-deterministic behavior becomes highest even with a temperature of 0.
+
+Due to the factors mentioned above, different answers may be returned even for the same question.
+
+**Workarounds:**
+1. Using math.SmallestNonzeroFloat32: By specifying math.SmallestNonzeroFloat32 in the temperature field instead of 0, you can mimic the behavior of setting it to 0.
```

Comment:
Might be useful to use code formatting here
```suggestion
1. Using `math.SmallestNonzeroFloat32`: By specifying `math.SmallestNonzeroFloat32` in the temperature field instead of 0, you can mimic the behavior of setting it to 0.
```

Reply:
@sashabaranov Your suggestion accepted. 👍